README.md

```diff
@@ -157,6 +161,34 @@ Using the above configuration, an error containing an `errorCode` property will
 }
 ```
 
+### Custom Renderer Functions
+
+You can switch out the default renderers for HTML, JSON, text and XML responses with custom renderers:
+
+```
+{
+  // ...
+  "final:after": {
+    "strong-error-handler": {
+      "params": {
+        "htmlRenderer": "$!./lib/send-html"
+      }
+    }
+}
+```
+
+The `$!` characters indicate that the path is relative to the location of `middleware.json`.
```

Comment:
Please add a note that `$!` prefix is a feature provided by LoopBack. When using strong-error-handler directly from JavaScript code, users should provide the actual function. (Please include an example code for that too.)